job_templates/py3-v4.2.1.sh

```diff
@@ -13,14 +13,16 @@ FINAL_OUT={final_out}
 KEEP_CRASHED_FILES={keep_crashed_files}
 
 
+cd
+echo 'PWD: '$(pwd)
 echo 'Starting job on Host: '$HOSTNAME
 echo 'Loading py3-v4.2.1'
 eval `/cvmfs/icecube.opensciencegrid.org/py3-v4.2.1/setup.sh`
-export PYTHONUSERBASE=/INSERT/PATH/HERE
+export PYTHONUSERBASE=/data/user/lneste/prompt_muons/py3-v4.2.1
 echo 'Using PYTHONUSERBASE: '${PYTHONUSERBASE}
 
 export PATH=$PYTHONUSERBASE/bin:$PATH
-export PYTHONPATH=$PYTHONUSERBASE/lib/python3.7/site-packages:$PYTHONPATH
+export PYTHONPATH=$PYTHONUSERBASE/lib/python3.10/site-packages:$PYTHONPATH
```

Comment:
Maybe we should figure out a more robust way here that reduces places in which modifications need to be made when changing versions